examples/tokenless/vite.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import { codecovVitePlugin } from "@codecov/vite-plugin";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    rollupOptions: {
+      output: {
+        assetFileNames: "[name].[hash].js",
+        chunkFileNames: "[name]-[hash].js",
+      },
+    },
+  },
+  plugins: [
+    react(),
+    codecovVitePlugin({
+      enableBundleAnalysis: true,
+      bundleName: "@codecov/example-tokenless-app",
+      uploadToken: process.env.TOKENLESS_UPLOAD_TOKEN,
```

Comment:
It is a little confusing to me that this tokenless example includes the token here - would it make sense to omit it?

Reply:
So tokenless only works on forked branches `forkOwner:branchName`, so when you're running it "locally" i.e. I make a PR and open it here, there's no `forkOwner` in the branch name so tokenless doesn't work. However when a user forks a repo and makes an upstream pull request the secrets aren't shared for obvious reason so that's when tokenless kicks in. It's a confusing thing that I still barely know anything about 😂

Reply:
Oooh okay I see! Got it - thanks for breaking that down!!

Reply:
(un-resolving so the thread is not hidden, good context for people)